package.json

```diff
@@ -5,17 +5,21 @@
   "scripts": {
     "build-dev": "webpack --config ./conf/webpack.conf.js --watch",
     "build": "webpack --config ./conf/webpack.prod.conf.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint ./public/**/**/*.js"
```

Comment:
should we add the `--fix` flag?

Reply:
I'm personally *really* not a fan of automated code fixing, just don't trust it! Worth a discussion though. Perhaps `--fix-dry-run`?

Reply:
We used it in MAM, the only issue we had was odd formatting (which you'll have to do to get the linter to pass anyway).

Let's ship this and iterate.